partials/hero.hbs

```diff
@@ -2,30 +2,39 @@
 If the template in question has a background image, then we render responsive image styles
 for it, and apply those styles to the <header> tag.
 --}}
+{{!--
+If the template in question has a background image, then we render responsive image styles
+for it, and apply those styles to the <header> tag.
+--}}
 
 {{#if background}}
   {{#contentFor "herobackground"}}
-  <style>
-    .m-hero__picture {
-      background-image: url({{img_url background size='xl'}});
-    }
-  
-    @media(max-width: 1000px) {
+    <style>
       .m-hero__picture {
-        background-image: url({{img_url background size='l'}});
+        display: flex;
```

Comment:
Please move these styles to the `.m-hero__picture` selector in the `src/sass/components/hero/_hero.scss` file.

Reply:
Done!